source/common/network/raw_buffer_socket.cc

```diff
@@ -16,6 +16,7 @@ void RawBufferSocket::setTransportSocketCallbacks(TransportSocketCallbacks& call
 IoResult RawBufferSocket::doRead(Buffer::Instance& buffer) {
   PostIoAction action = PostIoAction::KeepOpen;
   uint64_t bytes_read = 0;
+  bool didBlock = true;
```

Comment:
Is this supposed to be initialized to false and set to true when (result.err_->getErrorCode() == Api::IoError::IoErrorCode::Again)

Reply:
fixed in c7b20241c5eeed82d8d8ed356a8ac79d2d935c9c